rtptransceiver.go

```diff
@@ -8,7 +8,6 @@ import (
 
 // RTPTransceiver represents a combination of an RTPSender and an RTPReceiver that share a common mid.
 type RTPTransceiver struct {
-	Mid       string
```

Comment:
See https://github.com/pion/webrtc/pull/745/commits/5492a221456a8bdd4dce014d0512bd0c3bd0de0c for why this was removed